src/Symfony/Component/Yaml/README.md

```diff
@@ -1,21 +1,33 @@
 Yaml Component
 ==============
 
-YAML implements most of the YAML 1.2 specification.
+[![Package version](https://img.shields.io/packagist/v/symfony/yaml.svg)](https://packagist.org/packages/symfony/yaml)
 
-```php
-use Symfony\Component\Yaml\Yaml;
+The Yaml Component implements most of the
+[YAML 1.2 specification](http://yaml.org/spec/1.2/spec.html).
 
-$array = Yaml::parse(file_get_contents(filename));
+[See the Documentation](http://symfony.com/doc/current/components/yaml).
```

Comment:
@theofidry symfony.com does not yet enforce HTTPS. but it is best to start using it, as it is available since a few months